web3j/src/main/java/com/vmware/ethereum/config/AppConfig.java

```diff
@@ -103,6 +115,52 @@ public Credentials credentials() throws IOException, CipherException {
         credentials.getWalletPassword(), credentials.getWalletFile());
   }
 
+  @Bean
+  public TransactionReceiptProcessor queuedTransactionReceiptProcessor(
+      Web3j web3j, CountDownLatch countDownLatch, MetricsService metrics) {
+    Receipt receipt = config.getReceipt();
+    return new QueuingTransactionReceiptProcessor(
+        web3j,
+        new Callback() {
+          @Override
+          public void accept(TransactionReceipt transactionReceipt) {
+
+            log.debug("Receipt: {}", transactionReceipt);
+            Duration duration =
+                between(txHashTime.get(transactionReceipt.getTransactionHash()), now());
+            metrics.record(duration, transactionReceipt.getStatus());
+            countDownLatch.countDown();
+          }
+
+          @Override
+          public void exception(Exception exception) {
+            metrics.record(null, exception);
+            countDownLatch.countDown();
+          }
+        },
+        receipt.getAttempts(),
+        receipt.getInterval());
+  }
+
+  @Bean
+  public TransactionManager queuedTransactionManager(
```

Comment:
This entire method looks duplicate of transactionManager(..)

Reply:
Removed duplicate transactionManager code